content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,137 @@
+re-flashed with *Etcher* software to benefit from Odoo's latest :abbr:`IoT (Internet of Things)`
+image update. This means the Odoo :abbr:`IoT (Internet of Things)` box software may need to be
+updated in instances of a new :abbr:`IoT (Internet of Things)` box, or when a handler's update or
+an update from the :abbr:`IoT (Internet of Things)` box home page, do not resolve issues.
+
+
+.. note::
+   - It is often necessary to re-flash the :abbr:`IoT (Internet of Things)` box's image after
+     upgrading the Odoo database to a new version.
+   - A computer with a micro SD card reader/adapter is **required** to re-flash the micro SD card.
+
+First, begin by downloading `Etcher <https://www.balena.io/etcher#download-etcher>`_. It is a free,
+open-source utility, used for burning image files onto drives. After the download completes, install
+and launch the program on the computer.
+
+Then, download the latest :abbr:`IoT (Internet of Things)` image from `nightly
+<http://nightly.odoo.com/master/iotbox>`_, which will be labeled as `iotbox-latest.zip`. This
+particular image is compatible with *all* supported versions of Odoo.
+
+After this step is complete, insert the :abbr:`IoT (Internet of Things)` box's micro SD card into
+the computer or reader. Open *Etcher*, and select :guilabel:`Flash from file`, then find and select
+the `iotbox-latest.zip` image and extract it. Next, select the drive the image should be burned to.
+
+Lastly, click :guilabel:`Flash`, and wait for the process to finish.
+
+.. image:: updating_iot/etcher-app.png
+   :align: center
+   :alt: Balena's Etcher software dashboard.
+
+.. tip::
+   Balena's *Etcher* software also allows for the administrator to flash the :abbr:`SD (Secure
+   Digital)` card from a :abbr:`URL (Uniform Resource Locator)`. To flash from a :abbr:`URL (Uniform
+   Resource Locator)`, simply click :guilabel:`Flash from URL`, instead of :guilabel:`Flash from
+   file`.
+
+   Then, enter the following: `http://nightly.odoo.com/master/iotbox/iotbox-latest.zip`.
+
+   .. image:: updating_iot/url-flash.png
+      :align: center
+      :alt:  A view of Balena's Etcher software, with the flash from URL option highlighted.
+
+.. note::
+   An alternative software for flashing the micro SD card is `Raspberry Pi Imager
+   <https://www.raspberrypi.com/software/>`_.
+
+.. _iot/config/homepage-upgrade:
+
+Update from the IoT box home page
+=================================
+
+In the background, the :abbr:`IoT (Internet of Things)` box uses a version of Odoo code to run and
+connect to the Odoo database. This code may need to be updated in order for the :abbr:`IoT (Internet
+of Things)` box to operate effectively. This operation should be completed on a routine basis, to
+ensure the :abbr:`IoT (Internet of Things)` system, and its processes, stay up-to-date.
+
+Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to :menuselection:`IoT app -->
```

Comment:
```suggestion
Go to the :abbr:`IoT (Internet of Things)` box home page by navigating to :menuselection:`IoT app -->
```
you've used "home page" as two words multiple times up to this point, so I figured it'd be best to keep things uniform